### ISSUE TITLE
m1?

### ISSUE THREAD

**Issue by andrewhello** [Obbut/iCloud-Control#40]
it seems it is no longer support m1. :(( It is the best app ever.

**Comment by lawrencebensaid** [comment]
Agreed.
Maybe @Obbut could take a look.

**Comment by assolisc** [comment]
Same here, it would be great for it to have support for the latest macOS as well as natively for M1.

**Comment by fkick** [comment]
Hi All, this functionality to remove iCloud downloads is built into MacOS Catalina and Big Sur.

https://github.com/Obbut/iCloud-Control#a-note-about-macos-catalina

**Comment by Laesx** [comment]
> Hi All, this functionality to remove iCloud downloads is built into MacOS Catalina and Big Sur.
> 
> https://github.com/Obbut/iCloud-Control#a-note-about-macos-catalina

Nope, it's not fully, they explain in issue #39, you can't stop downloads or remove local downloads from subfolders unless EVERYTHING is downloaded, which is awful, I still have to use this app everyday, since the alleged "implementation" is useless.

**Comment by ghost** [comment]
> [...] you can't stop downloads or remove local downloads from subfolders unless EVERYTHING is downloaded

Yea this is such a poorly implemented feature. I don't understand why they did it like this.

**Comment by Njmcq** [comment]
Just managed to update the version of Swift within the Xcode project, make some minor adjustments to the code, and compile it for Apple silicon. Will do my best to patch some bugs and publish it as a new repository.

**Comment by Njmcq** [comment]
[https://github.com/Njmcq/iCloud-Control](https://github.com/Njmcq/iCloud-Control)

V1.3 is now Universal, making it compatible with both Intel- and Apple silicon-based Macs. Some minor bugs were also fixed.

**Comment by maniamax36** [comment]
Thank you! It really is nice to be able to cancel downloads that haven’t finished downloading yet.

**Comment by Obbut** [comment]
Hey! Thanks for the updates @Njmcq. I indeed haven't really been involved in this project for the last few years, but unlike what I expected I now see that it can still be very relevant for some users.

I'd be happy to accept any PRs and do some clean-up in the issues & pull requests of this repo 😉 

If you want to, feel free to get in touch with me (on Telegram: https://telegram.me/Obbut, or you can find my e-mail on my profile) and maybe we can have a chat to collaborate on this together.
